docs/excel/excel-add-ins-tables.md

```diff
@@ -353,6 +353,35 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+### AutoFilter
+
+> [!NOTE]
+> `Table.autoFilter` is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+As of the `ExcelApi 1.9` requirement set, add-ins have access to the table's [AutoFilter](/javascript/api/excel/excel.autofilter). An AutoFilter object is the entire filter structure of a table or range. All the aforementioned filter operations are compatible with the auto-filter. The single access point does make it easier to access and manage multiple filters.
+
+The following code sample shows the same [data filtering as the earlier code sample](#apply-filters-to-a-table), but done entirely through the auto-filter.
+
+```js
+    Excel.run(function (context) {
+        var sheet = context.workbook.worksheets.getItem("Sample");
+        var expensesTable = sheet.tables.getItem("ExpensesTable");
+
+        expensesTable.autoFilter.apply(expensesTable.getRange(), 2, {
+            filterOn: Excel.FilterOn.values,
+            values: ["Restaurant", "Groceries"]
+        });
+        expensesTable.autoFilter.apply(expensesTable.getRange(), 3, {
+            filterOn: Excel.FilterOn.dynamic,
+            dynamicCriteria: Excel.DynamicFilterCriteria.belowAverage
+        });
+
+        return context.sync();
+    }).catch(errorHandlerFunction);
+```
+
+An `AutoFilter` can also be applied to a range at the worksheet level. See the ["Filter data" section of Work with worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md#filter-data) for more information.
```

Comment:
Since the hyperlink points to the section, you might consider removing the text `"Filter data" section of "` from the link text (for brevity).

-> See [Work with worksheets using the Excel JavaScript API](excel-add-ins-worksheets.md#filter-data) for more information.